scripts/performance/benchmark

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+"""
+Use for benchmarking performance of other scripts. Provides data about
+time, memory use, cpu usage, network in, network out about the script ran in
+the form of a csv.
+
+
+Usage
+=====
+
+NOTE: Make sure you run ``pip install -r requirements.txt`` before running.
+
+To use the script, run::
+
+    ./benchmark "./my-script-to-run"
+
+
+If no ``--output-file`` was provided, the data will be saved to
+``performance.csv``
+"""
+import argparse
+import os
+import sys
+import subprocess
+import time
+
+import psutil
+
+
+# Determine the interface to track network IO depending on the platform.
+if sys.platform.startswith('linux'):
+    INTERFACE = 'eth0'
+elif sys.platform == 'darwin':
+    INTERFACE = 'en0'
+else:
+    raise RuntimeError('Script cannot be run on %s' % sys.platform)
```

Comment:
Yeah, the interface would be different though. It could actually be different if the user has a more complicated setup, such as if they're using en1 by default or if their interface has a different naming scheme.